free-programming-books-ta.md

```diff
@@ -43,3 +45,13 @@
 ### PHP
 
 * [எளிய தமிழில் PHP](https://freetamilebooks.com/ebooks/learn-php-in-tamil/)
+
+
+### Ruby
+
+* [எளிய தமிழில் Ruby](http://www.kaniyam.com/download/learn-ruby-in-tamil.pdf)
```

Comment:
Hello,

your resources fail to pass the Travis CI build because you need to indicate the format, PDF in this case. 

> Put a single space between the link and its format:

This is in reference to [CONTRIBUTING](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md) file under "Formatting."

```suggestion
* [எளிய தமிழில் Ruby](http://www.kaniyam.com/download/learn-ruby-in-tamil.pdf) (PDF)
```